index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+  integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+  crossorigin="anonymous"
+/>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+  <div id="pagina">
+    <header id="header">
+      <div id="header-img">
+        <img src="https://picsum.photos/150/100" alt="Logo">
+      </div>
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#home">Home</a></li>
+          <li ><a class="nav-link" href="#about">About</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="container">
+
+    <section id="hero">
+      <h2>Make your Bussines Grow!</h2>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input
+        name="email"
+        id="email"
+        type="email"
+        placeholder="Enter your email address"
+        required
+      />
+      <input id="submit" type="submit" value="Get Started" class="btn" />
+      </form>
+    </section>
+    
+    
+      <section id="home">
+        <div class="row">
```

Comment:
You should change the every **div class="row"** for an article tag. It's going to make your code semantic appropriately, clean and much easy to read.
Thanks 😃